infrastructure/rds.tf

```diff
@@ -24,10 +25,13 @@ resource "aws_db_instance" "db" {
 
   storage_type      = "gp2"
   allocated_storage = 20
+  skip_final_snapshot = true
 
   username = "root"
   password = random_id.postgres-garden-of-intelligence-password.hex
 
+  publicly_accessible = true
```

Comment:
modifying DB Instance garden-of-intelligence: InvalidVPCNetworkStateFault: Cannot create a publicly accessible DBInstance. The specified VPC has no internet gateway attached.Update the VPC and then try again